### DISCUSSION TITLE
What happened to the slack channel?

### DISCUSSION THREAD

**Discussion by FarhanSajid1** [micro/go-micro#2407] [Q&A] [answered]
I no longer have access to join the slack channel, and no links online point back to the slack channel

**Comment by asim** [comment, marked as answer]
Go Micro is now GitHub only

**Reply by FarhanSajid1** [reply]
That’s unfortunate, I thought it was a quick way to get some advise and feedback

**Reply by asim** [reply]
Unfortunately it was a support burden on me. 4000 to 1 in a slack is pretty bad. Go Micro is back to being a personal project and I only respond when I feel like it, otherwise a community of maintainers is forming around it.

**Reply by FarhanSajid1** [reply]
Yeah that sounds like a huge pain, thanks again for the response and the creation of this project
